packages/react/src/components/ActionMenu/ActionMenu.tsx

```diff
@@ -31,6 +33,12 @@ type ActionMenuProps = {
   trigger: React.ReactElement | ActionMenuTriggerFunction;
   /** Render the action menu in a different location in the dom. */
   portal?: React.RefObject<HTMLElement> | HTMLElement;
+  /**
+   * If true, the menu will will be passed as a children prop to the trigger function.
+   * If false, the menu will render as a sibling of the trigger.
+   * Only supported if trigger is a function *and* portal is undefined.
+   */
+  renderInTrigger?: boolean;
```

Comment:
This seems necessary, so do not take my comments below as saying that we should remove this for the problem we're trying to solve right now.

I do not think this should be a part of our "public" api, and want to make it clear, this should only remain as a short-term fix and work-around until we can address https://github.com/dequelabs/cauldron/issues/1934 or further investigate the focus problem that we are encountering. I don't think it's necessary to add this as a documented prop for the ActionMenu component, but we would want to include this in TopBar documentation as you've already done above.

Can we include in the tsdoc about specifically when this should be used?